pt/INSTALL.md

```diff
@@ -0,0 +1,19 @@
+## Installation instruction (conda)
+
+```
+# installation pytorch
+conda install pytorch torchvision cudatoolkit=10.1 -c pytorch-nightly
+
+# annoyingly this will need installation of old torchvision
+pip uninstall torchvision
+conda install av -c conda-forge
+pip install torchvision==0.5
+
+pip install submitit
+
+# arrgh
+pip install tensorflow
```

Comment:
why tensorflow?

Reply:
Because of the logging system - we're using tensorboard (which I found to be the best solution), but it requires full tensorboard install